common/persistence/visibility/store/elasticsearch/query_interceptors.go

```diff
@@ -120,19 +148,23 @@ func (ni *nameInterceptor) Name(name string, usage query.FieldNameUsage) (string
 }
 
 func (vi *valuesInterceptor) Values(fieldName string, values ...interface{}) ([]interface{}, error) {
-	fieldType, err := vi.searchAttributesTypeMap.GetType(fieldName)
+	name := fieldName
+	if _, exists := vi.nameInterceptor.fieldTransformations[fieldName]; exists {
+		name = vi.nameInterceptor.fieldTransformations[fieldName].newField
+	}
+
+	fieldType, err := vi.searchAttributesTypeMap.GetType(name)
```

Comment:
This is a map from field name to type, not alias to type.

Reply:
This code hasn't changed AFAIK